### PR TITLE
feat:create login view

### PR DESCRIPTION
**Pull request by LeVHoangduc**
@hungnguyenhuu-agilityio @lenguyenthimy-agilityio Please help me to check and merge it

### CODE REVIEW

File: src/javascripts/views/loginView.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      };
+
+      if (formValidator("user")(userCurrent)) {
+        const users = await getUsers();
+        let isUser = users.find((user) => {
+          return user.username === userCurrent.email && user.password === userCurrent.password;
+        });
+        isUser ? (window.location.href = "home.html") : alert(`${ERROR_MESSAGE.LOGIN_VALIDATION}`);
+      } else {
+        alert("Something maybe is wrong in your information!");
```
Comment (hungnguyenhuu-agilityio) [inline src/javascripts/views/loginView.js]: I saw you define this mess in constant, right? why don't you use it?

File: src/javascripts/views/loginView.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    this.loginForm = document.querySelector(".login-form");
+  }
+
+  //----- EVENT HANDLER -----//
+
+  /**
+   * Add event listener for login action to the add contact button.
+   * @param {callback} getUser
+   */
+  addEventLogin = (getUsers) => {
```
Comment (hungnguyenhuu-agilityio) [inline src/javascripts/views/loginView.js]: Can not get your idea about this function. Please help check it again

Comment (LeVHoangduc) [reply src/javascripts/views/loginView.js]: @hungnguyenhuu-agilityio I have updated it

File: src/javascripts/views/loginView.js (hunk 1 above)
Comment (LeVHoangduc) [reply src/javascripts/views/loginView.js]: @hungnguyenhuu-agilityio I have updated it

File: src/javascripts/views/loginView.js (hunk 2 above)
Comment (hungnguyenhuu-agilityio) [reply src/javascripts/views/loginView.js]: This is view. So the param should not be `getUsers`. The view just receive the data, not the logic.

**Comment by LeVHoangduc** [comment]
@hungnguyenhuu-agilityio I have updated my login view. Please help me to check again.

**Comment by LeVHoangduc** [comment]
@hungnguyenhuu-agilityio  I have updated my login view. Please help me to check it.

**Pull request merged**